ss_admin_console/main/tests/classes/testdata/TestClass.cls

```diff
@@ -1,2 +1,5 @@
 public abstract class TestClass {
+    public static Double calculate(Double x) {
```

Comment:
Why static?

Reply:
Static let us use this method at once like that: TestClass.calculate(2)